io.github.rinigus.PureMaps.json

```diff
@@ -0,0 +1,157 @@
+                }
+            ]
+        },
+        {
+            "name": "python3-pyxdg",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} pyxdg"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/47/6e/311d5f22e2b76381719b5d0c6e9dc39cd33999adae67db71d7279a6d70f4/pyxdg-0.26.tar.gz",
+                    "sha256": "fe2928d3f532ed32b39c32a482b54136fe766d19936afc96c8f00645f9da1a06"
+                }
+            ]
+        },
+        {
+            "name": "mapbox-gl-native-qt",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp mapbox-gl-native-lib.pro mapbox-gl-native",
+                "qmake mapbox-gl-native/mapbox-gl-native-lib.pro",
+                "make INSTALL_ROOT=/app install"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/rinigus/pkg-mapbox-gl-native.git",
+                    "branch": "sfos",
+                    "commit": "426263007b95480d30939648bd92bb6341f8395f"
+                }
+            ]
+        },
+        {
+            "name": "mapbox-gl-qml",
+            "buildsystem": "qmake",
+            "config-opts": [
+                "INCLUDEPATH+=/app/usr/include",
+                "LIBS+=-L/app/usr/lib"
+            ],
+            "make-install-args": [
+                "INSTALL_ROOT=/app"
+            ],
+            "post-install": [
+                "ln -s /app/usr/lib/qml/MapboxMap /app/lib/qml"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rinigus/mapbox-gl-qml/archive/1.3.2.tar.gz",
+                    "sha256":  "6222bc7981c78480f8318056f6733f4ec12f66a28f9f3f4058591dea9ddff92c"
+                }
+            ]
+        },
+        {
+            "name": "qmlrunner",
+            "buildsystem": "qmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/rinigus/qmlrunner.git",
+                    "commit": "d5b1dd3c4085adf1e8fd2e498cfacfc08b2da037"
+                }
+            ]
+        },
+        {
+            "name": "mimic",
+            "config-opts": [
+                "--with-audio=none"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/MycroftAI/mimic/archive/1.2.0.2.tar.gz",
+                    "sha256": "6adcc9911b09d6e9513add41ad9dfc0893ece277f556419869520a0f0708c102"
+                }
+            ]
+        },
+        {
+            "name": "pure-maps",
+            "buildsystem": "simple",
+            "build-commands": [
```

Comment:
Your build system should handle all of this in a single `make install`. It doesn't make sense to have package specific workarounds.

Reply:
I moved packaging commands into Pure Maps Makefile, in line with the used targets for Sailfish OS RPM generation. Similar to SFOS, I added a new target and handle the specifics over there. Corresponding commits are published